examples/seismic/tutorials/01_modelling.ipynb

```diff
@@ -37,7 +37,7 @@
     "\\end{cases}\n",
     "\n",
     "with the zero initial conditions to guarantee unicity of the solution.\n",
-    "The boundary conditions are Dirichlet conditions :\n",
+    "The boundary conditions are Dirichlet:\n",
```

Comment:
I preferred with repeated "conditions"/
Or: "The Dirichlet boundary conditions are:"

Reply:
Fixed